backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -31,9 +31,16 @@ import { UserRepository } from '../../typeorm/repository/User'
 export class UserResolver {
   @Query(() => User)
   @UseMiddleware(klicktippNewsletterStateMiddleware)
-  async login(@Args() { email, password }: UnsecureLoginArgs, @Ctx() context: any): Promise<User> {
+  async login(
+    @Args() { email, password, hasElopage }: UnsecureLoginArgs,
+    @Ctx() context: any,
+  ): Promise<User> {
     email = email.trim().toLowerCase()
-    const result = await apiPost(CONFIG.LOGIN_API_URL + 'unsecureLogin', { email, password })
+    const result = await apiPost(CONFIG.LOGIN_API_URL + 'unsecureLogin', {
+      email,
+      password,
+      hasElopage,
```

Comment:
I don't understand why we add this parameter to the unsecureLogin.
I think what @ulfgebhardt meant was to add hasElopage to the response of the unsecureLogin call, or have I understood it wrongly?

Reply:
No you are correct. What I was talking about was a return value not a argument to the login.

`Whenever an user logs in, he can retrieve his eloPage status as well`

The same applies to register I guess - `Whenever a new user registers, he can retrieve his eloPage status upon success` (this scenario is under the assumption that after a register, you are immediately logged in - i am not sure that is the case, tho)

Reply:
It seems Apollo is bad calling services with grapqhl similar functionality? :)